docs-2.0/7.data-security/4.ssl.md

```diff
@@ -0,0 +1,64 @@
+# SSL encryption
+
+Nebula Graph supports transmission with SSL encryption between the client, Graph service, Meta service, and Storage service. This topic describes how to enable SSL encryption.
+
+## Precaution
+
+Enabling SSL encryption will slightly affect performance.
+
+## Parameters
+
+| Parameter | Default value | Description |
+| ------------------ | ------- | ------------------------------|
+| `cert_path`        | -       | The path to the pem certification. |
+| `key_path`         | -       | The path to the key certification. |
+| `password_path`    | -       | The path to the password file certification. |
+| `ca_path`          | -       | The path to the trusted CA file. |
+| `enable_ssl`       | `false`   | Whether to enable SSL encryption. |
+| `enable_graph_ssl` | `false`   | Whether to enable SSL encryption in Graph service only. |
+| `enable_meta_ssl`  | `false`   | Whether to enable SSL encryption in Meta service only. |
+
+## Certificate modes
+
+To use SSL encryption, SSL certificates are required. Nebula Graph supports two certificate modes.
+
+- Self-signed certificate mode
+
+  In this mode, users need to make the signed certificate by themselves and set `cert_path`, `key_path`, and `password_path` in the corresponding file according to encryption policies.
+
+- CA-signed certificate mode
+
+  In this mode, users need to apply for the signed certificate from a certificate authority and set `cert_path`, `key_path`, and `password_path` in the corresponding file according to encryption policies.
+
+## Encryption policies
+
+Nebula Graph supports three encryption policies. For details, see [Usage explanation](https://github.com/vesoft-inc/nebula/blob/a67d166b284cae1b534bf8d19c936ee38bf12e29/docs/rfcs/0001-ssl-transportation.md#usage-explanation).
+
+- Encrypt the data transmission between the client, Graph service, Meta service, and Storage service.
+
+  The values of `nebula-graphd.conf`, `nebula-metad.conf`, and `nebula-storaged.conf` need to be modified. And `enable_ssl` should be set to `true`.
+
+- Encrypt the data transmission between the client and Graph service.
+  
+  This policy applies to the case that the clusters are set in the same room. Only the port of the Graph service is open to the outside because other services can communicate over the internal network without encryption. The value of `nebula-graphd.conf` needs to be modified. And `enable_graph_ssl` should be set to `true`.
```

Comment:
will in the same server room be better?

Reply:
[The value of `nebula-graphd.conf` needs to be modified. And `enable_graph_ssl` should be set to `true`.]

Add `enable_ssl = true` in the `nebula-graphd.conf` file.